source/develop/release-management/features/network/managed_mtu_for_vm_networks.html.md

```diff
@@ -0,0 +1,342 @@
+    link/ether 5e:fa:6f:20:b3:52 brd ff:ff:ff:ff:ff:ff
+37: ovirtmgmt: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1234 qdisc noqueue state UP mode DEFAULT qlen 1000
+    link/ether 52:54:00:4d:a1:11 brd ff:ff:ff:ff:ff:ff
+54: vnet0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1234 qdisc pfifo_fast master ovirtmgmt state UNKNOWN mode DEFAULT qlen 1000
+    link/ether fe:1a:4a:16:01:02 brd ff:ff:ff:ff:ff:ff
+55: vnet1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1100 qdisc pfifo_fast master ovs-system state UNKNOWN mode DEFAULT qlen 1000
+    link/ether fe:1a:4a:16:01:04 brd ff:ff:ff:ff:ff:ff
+[root@host23 ~]# ovs-vsctl show
+6861a627-a6c9-41dd-ab88-b0e0f74525b1
+    Bridge br-int
+        fail_mode: secure
+        Port "vnet1"
+            Interface "vnet1"
+        Port br-int
+            Interface br-int
+                type: internal
+        Port "ovn-a8086d-0"
+            Interface "ovn-a8086d-0"
+                type: geneve
+                options: {csum="true", key=flow, remote_ip="192.168.122.53"}
+    ovs_version: "2.9.0"
+[root@host23 ~]# ovs-vsctl get Interface br-int mtu
+1100
+~~~~
+
+## Prerequisites
+Setting the MTU via libvirt will work only on hosts with RHEL 7.4 or later.
+For hosts with RHEL 7.4 or later libvirt's XML is produced by engine, which will
+contain the definition. On hosts running older software, vdsm will create
+libvirt's XML without the MTU definition.
+Setting the MTU via libvirt works only on VMs with machine type rhel7.4.0 or
+later, so setting the MTU via libvirt will work transparent to the user if the
+default machine type in oVirt is rhel7.4.0 or later.
+
+Setting the MTU by DHCP individally per network will require an updated
+version of [OpenStack Java SDK][11] used by oVirt-engine.
+
+## Limitations
+
+* MTU is applied during the start of the VM, but not during an update of the
+  network.
+
+* Setting the MTU by DHCP requires that the relevant interface is configured
+  via DHCP by the guest.
+
+* Setting the MTU via libvirt requires that the machine type of the VM is
+  rhel7.4.0 or later, which requires that the hosts are running RHEL 7.4 or
+  later.
+
+* Setting the MTU via libvirt requires that vNIC interface is of model type
+  'virtio' and the VirtIO driver of the guest supports MTU negation.
+  This is given for RHEL 7.4 or later and VirtIO-Win version 1.9.1-0 (netkvm
+  build 139) or later, which is already given for the already released
+  oVirt-toolsSetup 4.2-1.
+
+## Benefit to oVirt
+
+If oVirt assist to configure the MTU inside the guest VM helps to get the VM
+working as intended by the user.
+
+If the MTU inside the guest VM is set too large, too large Ethernet frames sent
+by the VM are dropped without any hint to the user.
+If the MTU inside the guest VM is set too small, the performance of the vNIC
+might be below the users expectation.
+
+## Entity Description
+
+The MTU property has to be enabled for creating external networks in webadmin,
```

Comment:
The MTU property is already enabled by default to all new networks(1500), which means that you need to enable the custom MTU property.

Reply:
The UI is lying currently, and it this feature page suggest to lie in the future too.
Currently no MTU is defined for external networks, and ovirt-provider-ovn uses OVN's internal DHCP server to propagate a fixed configured value, see https://github.com/oVirt/ovirt-provider-ovn/blob/master/README.adoc#section-dhcp